packages/terra-search-field/tests/nightwatch/search-field-spec.js

```diff
@@ -74,4 +75,15 @@ module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous
       });
     });
   },
+
+  'Displays the search button disabled': (browser) => {
+    browser.url(`${browser.launchUrl}/#/tests/search-field-tests/disabled`);
+
+    browser.assert.expect.element('#disabledsearchfield').to.be.present;
```

Comment:
Remove the asserts here.

Reply:
Yeah... :( I was getting some other really weird errors and I assumed it was something windows related, but now realized that I goofed.